lib/src/channel.dart

```diff
@@ -1,6 +1,10 @@
 import 'package:nyxx/nyxx.dart';
+import 'package:nyxx_extensions/src/utils/formatters.dart';
 
-extension ChannelUtils on Channel {
+extension ChannelUtils on PartialChannel {
```

Comment:
This change will make the URL getter no longer function as the guild ID will not be included for partial guild channels.

Reply:
My bad,missed that